r2r/main/api/routes/ingestion.py

```diff
@@ -17,20 +12,6 @@ def __init__(self, engine: R2REngine):
         self.setup_routes()
 
     def setup_routes(self):
-        @self.router.post("/ingest_documents")
```

Comment:
Should we leave these the router just in case someone is using in production? Or have these endpoints reroute to ingest_files with a message?

Reply:
This is a good point, I was thinking to label it as a breaking change, but it might be nice to do something more progressive like marking this endpoint as deprecated and then removing next release. Wdyt?